### PR TITLE
add maildev starter

### PR DESCRIPTION
**Pull request by mittalyashu**

### CODE REVIEW

**Comment by mittalyashu** [comment]
@FarazPatankar what about the icon?

File: examples/maildev/README.md [hunk 1]
```diff
@@ -0,0 +1,13 @@
+---
+title: Maildev
+description: A simplest way to test your generated emails during development with an easy to use web interface.
+buttonSource: https://github.com/mittalyashu/railway-maildev/blob/main/readme.md
+tags:
+  - maildev
+  - nodejs
```
Comment (FarazPatankar) [inline examples/maildev/README.md:7]: Could you change this to `javascript` instead of `nodejs`? That's the pattern we have at the moment.

**Review by FarazPatankar — CHANGES_REQUESTED** [review]

**Comment by FarazPatankar** [comment]
@mittalyashu you can PR the icon here: https://github.com/railwayapp/devicons

File: examples/maildev/README.md (hunk 1 above)
Comment (mittalyashu) [reply examples/maildev/README.md:7]: sure

**Comment by mittalyashu** [comment]
> @mittalyashu you can PR the icon here: https://github.com/railwayapp/devicons

@FarazPatankar Maildev project doesn't seems have any logo, instead they're using this [icon](https://fontawesome.com/v5.15/icons/envelope?style=solid).

**Comment by mittalyashu** [comment]
@FarazPatankar Here's the link to maildev icon https://github.com/railwayapp/devicons/pull/18

**Review by coffee-cup — APPROVED** [review]

**Pull request merged**